### DISCUSSION TITLE
Change files' folder when updating metadata

### DISCUSSION THREAD

**Discussion by adepiero** [photoprism/photoprism#3923] [Ideas]
I just started to use PhotoPrism and imported about 54.000 files. About 100 of them are old or converted videos (DVDs), without "date taken" metadata. I let and I want PhotoPrism to move the files by itself.
I want to use PhotoPrism as my primary and only library tool, I don't want to use external tools.
I imported them as they were, changing metadata later. I do know that PhotoPrism does not update metadata back to the originals and it does not matter to me (even if it would be a great feature). There is still an issue, anyway: the files are not moved accordingly to the new metadata.
I expect them to be moved to the /originals/year/month folder taken from the "date" metadata, but they remain in the folder assigned them with the first indexing. So I do have some files with wrong internal metadata, stored in the wrong year/month folder.
Is there any plan to fix it?
Thank you.

**Comment by graciousgrey** [comment]
Both writing metadata back to original files and moving previously imported files within the originals folder if the date has been changed are planned for the long term. However, both of these are complex tasks and we cannot yet tell you when we will get to them.

Until then, you would have to move the files directly in your file system, PhotoPrism will then recognize the changed location of the files the next time you index them.
